src/components/Product/Product.js

```diff
@@ -16,6 +16,7 @@ const Product = product => {
         <span>{product.package}</span>
       </div>
     </div>
+    <button type="submit">Add</button>
```

Comment:
Yeah! I think breaking the button out into its own component would be really cool (and reusable) but I think that could be its own trello card to do for later.